### ISSUE TITLE
Pager decoder

### ISSUE THREAD

**Issue by choupette-change** [f4exb/sdrangel#1794]
If we put several pager decoders (with "duplicate channel") on the same workspace and we adjust the frequency, the decoding of the messages of the other 2 "pager decoders", in my case, seem capricious?? What am I doing wrong? The signals decoded in "Waves forms" are ok for the 3 frequencies??
Best regards
![image](https://github.com/f4exb/sdrangel/assets/68333247/67386742-2900-4e6c-a2c1-574740278729)

**Comment by srcejon** [comment]
Can you decode each channel ok just using a single demod?

**Comment by choupette-change** [comment]
Yes, if I take the decoders one by one it works. I close and reopen Sdrangel between each frequency test.

![image](https://github.com/f4exb/sdrangel/assets/68333247/fcba75ef-b6ff-4d46-8ed0-4413e419b470)

**Comment by srcejon** [comment]
Ok - can you please record an IQ .wav file so I can give it a try? Thanks.

**Comment by choupette-change** [comment]
https://drive.google.com/file/d/1FbggiGkQ1Mis31ueqaxpSR3mY07x6C62/view?usp=drive_link

**Comment by srcejon** [comment]
Thanks.

It seems it's only a problem if the "Duplicate channel" button is used. If I use "Add channel"  3 times, then it works.

**Comment by srcejon** [comment]
Please ignore the last message - I made a mistake.

If I replay your IQ file with 3 demods, I get:

![image](https://github.com/f4exb/sdrangel/assets/57259258/94b2a444-ab56-4959-bea7-2b7e847f857e)

What do you get? What do you think is wrong?

**Comment by choupette-change** [comment]
I still have the same problem, I don't understand why? I lowered the gain to avoid saturating the receiver (I am near the transmitter)?? Can you send me your settings files to test them with my SDRangel?
![image](https://github.com/f4exb/sdrangel/assets/68333247/b22e224d-bc4d-44e1-9ca6-d79b28b8cdc5)

**Comment by srcejon** [comment]
Ok, playing around some more, it seems as though there's a bug. Sometimes it works, sometimes it doesn't - even though the settings appear the same in the GUI. Not sure why yet.

**Comment by choupette-change** [comment]
Absolutely, I also noticed the random operation of this (these) decoders. Best regards

**Comment by srcejon** [comment]
Should be fixed by the above patch.
